Plugins/ExaTrkX/include/Acts/Plugins/ExaTrkX/ExaTrkXTrackFinding.hpp

```diff
@@ -0,0 +1,77 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <core/session/onnxruntime_cxx_api.h>
+
+namespace Acts {
+
+/// Class implementing the Exa.TrkX track finding algorithm.
+/// It holds the required ONNX objects.
+class ExaTrkXTrackFinding {
```

Comment:
We talked about it, however, we thought that it is enough to use a common interface on the algorithm level for the moment. So that other solutions would implement an algorithm that has the same input/output whiteboard entries...

But maybe @Corentin-Allaire @xju2 could also comment on this question...

Reply:
Yes, that is exactly what we wanted.